.eslintrc

```diff
@@ -38,6 +38,7 @@
         prefer-const: 1,
         no-const-assign: 2,
         "@typescript-eslint/no-explicit-any": 0,
+        "@typescript-eslint/explicit-function-return-type": 0,
```

Comment:
Would be good to discuss this as a team, I know it's slithy annoying, but I actually like it as a rule.

Reply:
I am open to the discussion. What I find annoying about this rule is that it impacts tights and obvious arrow-functions. For example, it requires something like this:

```
  return values.reduce((memo: number, i: number) => memo + i) / values.length;
```
to be (with prettier):
```
  return (
    values.reduce((memo: number, i: number): number => memo + i) / values.length
  );```

which is not a win, IMHO.

Reply:
We have been living without this rule in a separate repo effectively. TS inference is good enough that any issues are typically caught without explicit return type. Adding return type as needed for trickier functions to ensure the body of the function does what the author wants is a nice opt-in. If you use TS `strict` rules, this rule is less impactful.

Reply:
will leave this rule out for now and see how much it continues to irritate us (by which I mean "me" 😉)